integrations-and-supported-tools/transformers/scripts/run_examples.sh

```diff
@@ -0,0 +1,9 @@
+echo "Installing requirements..."
+pip install -r requirements.txt
+
+echo "Setting Neptune credentials..."
+export NEPTUNE_API_TOKEN=ANONYMOUS
+export NEPTUNE_PROJECT=common/huggingface-integration
+
+echo "Running Neptune_Transformers.py..."
+python Neptune_Transformers.py
```

Comment:
@kshitij12345 
`Neptune_Transformers_report_to.py` has to be added here as well

Reply:
Ah. Missed that. Thanks!